velox/duckdb/conversion/DuckParser.h

```diff
@@ -30,6 +30,9 @@ struct ParseOptions {
   // Retain legacy behavior by default.
   bool parseDecimalAsDouble = true;
   bool parseIntegerAsBigint = true;
+
+  // The prefix of registered functions.
+  std::string functionPrefix = "";
```

Comment:
I'm not sure I understand this change. Would you clarify. Changes to DuckParser need to be submitted in a separate PR with a test.

Reply:
Previously, DuckParser generates `in` by default which goes into presto sql function, when spark sql functions were registered with prefix `spark`. Here, I added the prefix as an option to let the user can control the generated function.

Reply:
Opened https://github.com/facebookincubator/velox/pull/5212 for this change. If the idea is OK, I can apply function prefix on all the needed places in DuckParser.